code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -413,7 +413,7 @@ There are several things that need to be remembered:
 			icon_file = dna.species.generate_custom_worn_icon(LOADOUT_ITEM_HEAD, head)
 			if(icon_file)
 				mutant_override = TRUE
-		if(!icon_file && (dna.species.bodytype & BODYTYPE_SNOUTED) && (worn_item.supports_variations_flags & CLOTHING_SNOUTED_VARIATION))
+		if((dna.species.bodytype & BODYTYPE_SNOUTED) && (worn_item.supports_variations_flags & CLOTHING_SNOUTED_VARIATION))
```

Comment:
This would be the proper fix, otherwise `BODYTYPE_CUSTOM` would lose its precedence over the snouted bodytype.
```suggestion
		if((!icon_file || icon_file == 'icons/mob/clothing/head/default.dmi') && (dna.species.bodytype & BODYTYPE_SNOUTED) && (worn_item.supports_variations_flags & CLOTHING_SNOUTED_VARIATION))
```